locations/spiders/sunbelt_rentals_us_ca.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+import json
+import re
+
+import scrapy
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class SunbeltRentalsUsCaSpider(scrapy.Spider):
+    name = "sunbelt_rentals_us_ca"
+    item_attributes = {"brand": "Sunbelt Rentals", "brand_wikidata": "Q102396721"}
+    allowed_domains = ["sunbeltrentals.com"]
+
+    def start_requests(self):
+        yield scrapy.Request(
+            "https://api.sunbeltrentals.com/web/api/v1/locations?latitude=39.0171368&longitude=-94.5985613&distance=5000",
+            headers={
+                "Client_id": "0oa56ipgl8SAfB1kE5d7",
+                "Client_secret": "6yNzPOIbav3xJ0XMFRI9cCKjEmqcKXiPVPhQS7eo",
+                "Logintoken": "",
+                "Request-Id": "12323sunbelt",
+                "X-Correlation-Id": "qjElWEoUHtD5oK0uqMbCivbwL632fc23bef67923b234f8c19",
+                "Origin": "https://www.sunbeltrentals.com",
+                "Referer": "https://www.sunbeltrentals.com/",
+                "Sec-Ch-Ua-Platform": "macOS",
+                "Authorization": "Bearer",
+                "Channel": "WEBAPP",
+                "Companyid": 0,
+                "Host": "api.sunbeltrentals.com",
+            },
+            method="GET",
+            callback=self.parse,
+        )
+
+    def parse_hours(self, hours):
+        opening_hours = OpeningHours()
+
+        for hour in hours:
+            day = hour["day"][:2]
+            if hour["hours"] == "Closed":
+                pass
```

Comment:
`continue` instead of `pass`? Then remove the `else` statement beneath? This would make the closed check a guard statement, minimising indentation and complexity of the code.